_install-and-configure/os-comp.md

```diff
@@ -27,6 +27,6 @@ The following table details changes made to operating system compatibility.
 
 | Date       | Issue | PR | Details |
 |:-----------|:-------|:-------|:--------------------------|
-| 2024-06-03 | [opensearch-build Issue 4379](https://github.com/opensearch-project/documentation-website/issues/4379) | [PR 7285](https://github.com/opensearch-project/documentation-website/pull/7285) | Remove CentOS7 as it if approaching [EOL](https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/) |
+| 2024-07-23 | [opensearch-build Issue 4379](https://github.com/opensearch-project/documentation-website/issues/4379) | [PR 7821](https://github.com/opensearch-project/documentation-website/pull/7821) | Remove CentOS7 as it if approaching [EOL](https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/) |
```

Comment:
Issue still wrong.
It should be opensearch-build issue 4379, not docs website.
https://github.com/opensearch-project/opensearch-build/issues/4379

Thanks!